src/components/page/MembersPage.tsx

```diff
@@ -8,10 +10,24 @@ interface IMembersPageProps {
 export const MembersPage = async ({ id }: IMembersPageProps) => {
   const users: IUser[] = await fetchUserList(id);
   return (
-    <>
-      {users.map((user, i) => (
-        <div key={i}> {user.name} </div>
-      ))}
-    </>
+    <div className="px-4">
+      <div className="flex items-center mt-[35px] mb-5">
+        <h2 className="text-xl mr-44">Members</h2>
+        <div className="w-12 h-12 flex justify-center items-center">
+          <Icon icon={MenuMeatballIcon} className="w-5 h-1" />
+        </div>
+        <div className="w-12 h-12 flex items-center justify-center">
+          <Icon icon={PlusIcon} className="w-[1.125rem] h-[1.125rem]" />
+        </div>
+      </div>
+      <div className="flex flex-col gap-2">
+        {users.map((user, i) => (
+          <Card key={i} className="flex h-[3.75rem] items-center">
```

Comment:
Please add "asChild" setting to `Card` component and rap its child component with `button` component.
If you do so, `Card` component will change from `<div>` to `<button>`, so that users can select and move each card component with tab button like this video.


https://github.com/genesis-tech-tribe/nishiki-frontend/assets/25422441/6c960fb0-7009-44d6-98ee-f9d97d8c9a13



```tsx
<Card key={i} asChild>
  <button className="flex gap-4 items-center h-[3.75rem]">
    <Icon icon={PersonCircleIcon} color="gray" size={11} className="pl-1" />
    <div className="ml-3">{user.name}</div>
  </button>
</Card>
```

Reply:
Hi, Kota!
Thanks for your review and comment! understand I need to set asChild attributes? and already done, with wrapping `button` element too.
I would like to confirm with you one thing about your code above. Do I need to set `gap-4` in `button` ?? in Figma, the gap between icon and name is 12px which is 3rem,  so I thought I should set `gap-3`(I need to delete `ml-3` for `user.name` `div`  in this case though)

<img width="221" alt="image" src="https://github.com/genesis-tech-tribe/nishiki-frontend/assets/142355969/f38af2cc-950c-4579-9ae4-2f131526f122">

Please kindly let me know.

Thanks!

Reply:
> I thought I should set gap-3(I need to delete ml-3 for user.name div in this case though)

Thank you for checking!
I agree with your thought. and I suppose no div for `{user.name}` and `pl-1` are also not needed.

Reply:
Thanks for quick check and response,
> I suppose div for {user.name} and pl-1 are not needed neither.

Yup! you are right. I modified them!

Thanks